pkg/observability/observability.go

```diff
@@ -12,13 +15,46 @@ const (
 )
 
 var (
-	RealmTagKey tag.Key
+	BuildIDTagKey  = tag.MustNewKey("build_id")
+	BuildTagTagKey = tag.MustNewKey("build_tag")
+	RealmTagKey    = tag.MustNewKey("realm")
 )
 
-func init() {
-	var err error
-	RealmTagKey, err = tag.NewKey("realm")
+// CommonTagKeys returns the slice of common tag keys that should used in all
+// views.
+func CommonTagKeys() []tag.Key {
+	return []tag.Key{
+		BuildIDTagKey,
+		BuildTagTagKey,
+		RealmTagKey,
```

Comment:
We should also include the revision name (from env var `K_REVISION`). A redeployment (e.g. changing an environment variable) would change the revision name.

Reply:
Done in a later PR